LibreHardwareMonitorLib/Interop/Kernel32.cs

```diff
@@ -206,7 +206,7 @@ internal static SafeHandle OpenDevice(string devicePath)
         [DllImport(DllName, SetLastError = true)]
         internal static extern void RtlZeroMemory(IntPtr Destination, int Length);
 
-        [DllImport(DllName, EntryPoint = "CopyMemory", SetLastError = false)]
+        [DllImport(DllName, EntryPoint = "RtlCopyMemory", SetLastError = false)]
         internal static extern void CopyMemory(IntPtr Destination, IntPtr Source, uint Length);
```

Comment:
What about simply renaming this to RtlCopyMemory? 😊

Reply:
Sounds good! Switched.